src/data/validation/email.js

```diff
@@ -90,8 +117,9 @@ const validateEmailAddressesFields = (formData) => {
     _error: [],
   };
 
-  const textAreaEmails = formData[EMAIL_ADDRESS_TEXT_FORM_DATA] && formData[EMAIL_ADDRESS_TEXT_FORM_DATA].split(/\r\n|\n/);
-  const csvEmails = formData[EMAIL_ADDRESS_CSV_FORM_DATA];
+  const extractedEmails = extractEmails(formData);
+  const textAreaEmails = extractedEmails.textEmails;
+  const csvEmails = extractedEmails.CSVEmails;
   let {
     invalidEmailIndices,
   } = validateEmailAddresses(textAreaEmails || csvEmails);
```

Comment:
[curious/clarification] Do we only support textarea _OR_ CSV upload? Aka, you can't provide textarea input _AND_ CSV upload together? This logic here would imply that it's one or the other, but I wonder if that's actually what we should be doing as a product requirement. Might be worth clarifying (not blocking/related to this PR, though!).

Reply:
In existing logic, On the UI we say that either input emails in textarea or upload csv but in reality we pick all the emails from both textarea and csv and then send them to backend for license assignment.

Reply:
Thanks for clarifying!